specification/mediaservices/resource-manager/Microsoft.Media/preview/2019-09-01-preview/MediaGraphs.json

```diff
@@ -0,0 +1,757 @@
+            }
+          },
+          "default": {
+            "description": "Detailed error information.",
+            "schema": {
+              "$ref": "#/definitions/ApiError"
+            }
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/MediaServicesAccountName"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "name": "$top",
+            "in": "query",
+            "required": false,
+            "type": "integer",
+            "description": "Specifies a non-negative integer n that limits the number of items returned from a collection. The service returns the number of available items up to but not greater than the specified value n."
+          }
+        ],
+        "x-ms-pageable": {
+          "nextLinkName": "@odata.nextLink"
+        },
+        "x-ms-examples": {
+          "List all Media Graphs": {
+            "$ref": "examples/media-graph-list-all.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Media/mediaServices/{accountName}/mediaGraphs/{mediaGraphName}": {
+      "get": {
+        "summary": "Get a Media Graph",
+        "description": "Get the details of a Media Graph in the Media Services account",
+        "operationId": "MediaGraphs_Get",
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/MediaGraph"
+            }
+          },
+          "404": {
+            "description": "NotFound"
+          },
+          "default": {
+            "description": "Detailed error information.",
+            "schema": {
+              "$ref": "#/definitions/ApiError"
+            }
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/MediaServicesAccountName"
+          },
+          {
+            "name": "mediaGraphName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The Media Graph name."
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          }
+        ],
+        "x-ms-examples": {
+          "Get a Media Graph by name": {
+            "$ref": "examples/media-graph-get-by-name.json"
+          }
+        }
+      },
+      "put": {
+        "summary": "Create or update a Media Graph",
+        "description": "Create or update a Media Graph in the Media Services account",
+        "operationId": "MediaGraphs_CreateOrUpdate",
```

Comment:
Is `creareOrUpdate` a long running operation? same question for `delete`.

Reply:
Both PUT (CreateOrUpdate) and DELETE are not long running. They complete synchronously. 
In our API, START and STOP actions are Long running operations